devfile.yaml

```diff
@@ -0,0 +1,48 @@
+metadata:
+  generateName: kubernetes-image-puller-operator-
+projects:
+  - name: kubernetes-image-puller-operator
+    source:
+      location: 'https://github.com/che-incubator/kubernetes-image-puller-operator.git'
+      type: git
+      branch: master
+attributes:
+  persistVolumes: 'false'
```

Comment:
do we really need to have ephemeral mode enabled? PVs on Hosted Che now has 2GB of storage

Reply:
Good catch, we do not.  Changed